src/lib/components/Header.svelte

```diff
@@ -0,0 +1,95 @@
+<script type="ts">
+  import { NAV_ELEMENTS } from "$lib/config/homePageData";
+
+  import { fly } from "svelte/transition";
+
+  import AnimatedHumburger from "./AnimatedHumburger.svelte";
+
+  export let open = false;
+  export let onClick = (): void => {
+    open = !open;
+
+    if (open) {
+      document.body.style.overflow = "hidden";
+    } else {
+      document.body.style.overflow = "scroll";
+    }
+  };
+</script>
+
+<header>
+  <section class="container">
+    <div class="desktop-menu surface-tint header">
+      <a href="/" class="on-secondary-text">EME STUDIO</a>
+      <nav>
+        <ul>
+          {#each NAV_ELEMENTS as navElements}
+            <li>
+              <a class="on-secondary-text" href="/">{navElements.name}</a>
+            </li>
+          {/each}
+        </ul>
+      </nav>
+      <a href="/home"
+        ><button class="background on-background-text label-large"
```

Comment:
You are putting innecessary code here..
Nesting a `button` inside an `a` tag doesn't make sense..
If you want an `a` tag that looks like a `button` maybe you can create a `.button` class that shares styles with the `button` element, and use it when you have this kind of situation..